packages.config

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="OSPSuite.Assets" version="11.2.155"/>
-  <package id="OSPSuite.Core" version="11.2.155"/>
-  <package id="OSPSuite.Infrastructure" version="11.2.155"/>
-  <package id="OSPSuite.Infrastructure.Import" version="11.2.155"/>
-  <package id="OSPSuite.Infrastructure.Autofac" version="11.2.155"/>
-  <package id="OSPSuite.R" version="11.2.155"/>
+  <package id="OSPSuite.Assets" version="12.0.218"/>
+  <package id="OSPSuite.Core" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure.Import" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure.Autofac" version="12.0.218"/>
+  <package id="OSPSuite.R" version="12.0.218"/>

```

Comment:
let's use the latest **12.0.226** 😉

Reply:
Is it already used by PK-Sim?

Reply:
PK-Sim currently uses 12.0.223 
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/0b33736c8b7efe83c2faf743fd12a54587d3e4bb/src/PKSim.Core/PKSim.Core.csproj#L31-L39

Reply:
Updated to 223